File: Célia_LingProgram/src/Lista03/Exer01_Tabuada.java
package Lista03;

import java.util.Scanner;

public class Exer01_Tabuada {
    public static void main(String[] args) {

        Scanner leitor = new Scanner(System.in);

        System.out.println("TABUADA");
        System.out.println("Insira um número:");
        Integer numero = leitor.nextInt();

        System.out.println("Tabuada do "+ numero);
        for (int i = 0; i <= 10; i++) {
            Integer result = i * numero;
            System.out.println(numero + " x " + i + " = " + result);
        }

    }
}

File: Célia_LingProgram/JavaFacul/src/Lista03/Exer01_Tabuada.kt
package Lista03

import java.util.*

//Crie uma classe chamada “Tabuada” que:
//1. Solicite um número para o usuário.
//2. Exiba a tabuada correspondente ao número digitado.
//Exemplo:
//2
//2 x 0 = 0
//2 x 1 = 2
//2 x 2 = 4
//2 x 3 = 6
//2 x 4 = 8
//2 x 5 = 10
//2 x 6 = 12
//2 x 7 = 14
//2 x 8 = 16
//2 x 9 = 18
//2 x 10 = 20

fun main() {

     val leitor = Scanner(System.`in`)

    println("Insira um número: ")
    val numero: Int = leitor.nextInt()

    for(contador in 1..10){
        println("$numero X $contador  = ${numero*contador}")
    }
}